build-files/mlst/2.23.0-2024-12-31/Dockerfile

```diff
@@ -0,0 +1,118 @@
+FROM ubuntu:jammy AS app
```

Comment:
maybe it's not worth the upgrade, we might consider using ubuntu:focal or ubuntu:noble

I fear that may break things though, since perl dependencies can be finicky.